logger.gd

```diff
@@ -220,7 +272,8 @@ const MAX_STRATEGY = STRATEGY_MEMORY*2 - 1
 const FORMAT_IDS = {
   "level": "{LVL}",
   "module": "{MOD}",
-  "message": "{MSG}"
+	"message": "{MSG}",
```

Comment:
Indentation issue.

Reply:
Still not fixed.

Reply:
I can't push to his repro and fix it but I would reformat the whole file according to #12 in a separate PR.
If  that is acceptable ?

Reply:
Whoops! Sorry, life happened etc and I didn't get back to this. Turns out it was a mix of tabs and spaces 🤮, but now it's aligned with the rest of the file (tabs).

Should be resolved now 👍